app/src/main/res/values/strings.xml

```diff
@@ -49,8 +49,45 @@
     <!-- View Record List screen strings -->
     <string name="edit_records">Edit</string>
   
-    <!-- View Problem screen stings-->
+    <!-- View Problem screen strings-->
     <string name="view_records">View Records</string>
     <string name="create_record">Create Record</string>
 
+    <!-- View Patient Edit Profile screen strings-->
+    <string name="pEditprofile_verify_prompt_text">Enter current password to save ALL changes made to profile information:</string>
```

Comment:
The strings for pEdit/pView and cEdit/cView looks the same, I don't think it needs to be split just have editprofile/viewprofile instead since we can use these strings more than once

Reply:
aiight, I'll just make it generic then.